src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -172,6 +174,32 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 		}));
 	}
 
+	private addContentTabGroup(): void {
```

Comment:
Why not just have the tab groups be defined in the JSON instead? If we want to add more groups this just gets messier and messier. And being able to define any arbitrary amount of tab groups seems like the direction we want to head anyways (and it's just an implementation that the dashboard for now is going to have two groups - contents and extensions)

Reply:
good point, i have made it more extensible with the latest commit.